src/reynir_correct/main.py

```diff
@@ -357,7 +386,7 @@ def main() -> None:
         options["apply_suggestions"] = True
 
     if args.grammar:
-        # Check grammar, output a JSON object for each sentence
+        # Check grammar, output a text or JSON object for each sentence
```

Comment:
So the --text option only works with --grammar?  In that case, we might just have an independent option that is not used with --grammar.

Reply:
The default for check_spelling() is returning text, so it is only needed here.